tasks/eva_2124/tests/test_correct_assembly_for_study.py

```diff
@@ -0,0 +1,130 @@
+            'tax': 9685,
+            'study': 'PRJEB30080',
+            'contig': 'B1',
+            'start': 150039652,
+            'ref': 'C',
+            'alt': 'T',
+            'accession': 5015497292,
+            'version': 1,
+            'createdDate': datetime.fromisoformat('2018-12-05T14:18:37.842')
+        }
+        horse_doc = {
+            '_id': '0000185B0438E7CB2DD5E8F9F2B62E89C398FDA4',
+            'seq': 'GCA_000002305.1',
+            'tax': 9796,
+            'study': 'PRJEB9799',
+            'contig': '1',
+            'start': 4573797,
+            'ref': 'G',
+            'alt': 'A',
+            'accession': 5000016398,
+            'version': 1,
+            'createdDate': datetime.fromisoformat('2018-06-20T08:31:07.900')
+        }
+        another_horse_doc = {
+            '_id' : 'D81824774C9D92D71971A970567FF9C5F510D283',
+            'seq' : 'GCA_000002305.1',
+            'tax' : 9796,
+            'study' : 'PRJEB27771',
+            'contig' : '21',
+            'start' : 30681601,
+            'ref' : 'C',
+            'alt' : 'T',
+            'accession' : 5081976263,
+            'version' : 1,
+            'createdDate' : datetime.fromisoformat('2018-09-20T14:31:26.164')
+        }
+
+        horse_already_genbank = {
+            '_id' : 'BB6C2E844664691DC42FC46456448833A8A35763',
+            'seq' : 'GCA_000002305.1',
+            'tax' : 9796,
+            'study' : 'PRJEB24630',
+            'contig' : 'CM000387.2',
+            'start' : 15520392,
+            'ref' : 'C',
+            'alt' : 'T',
+            'accession' : 5201460111,
+            'version' : 1,
+            'createdDate' : datetime.fromisoformat('2019-07-08T03:12:59.352')
+        }
+        self.connection_handle = get_mongo_connection_handle(username=self.username, password=self.password,
+                                                             host=self.host, authentication_database=self.admin_db)
+        self.connection_handle[self.eva_database][self.variant_collection].drop()
+        self.connection_handle[self.eva_database][self.variant_collection].insert_many([
+            original_document, horse_doc, another_horse_doc, horse_already_genbank])
+
+    def tearDown(self) -> None:
+        self.connection_handle[self.eva_database][self.variant_collection].drop()
+        self.connection_handle[self.admin_db].command("dropUser", self.username)
+        self.connection_handle.close()
+
+    def test_correct(self):
+        fixed = correct(self.username, self.password, self.host, studies=['PRJEB30080'],
+                        assembly_accession='GCA_000181335.3')
+        self.assertEqual(fixed, 1)
+        variant = (self.connection_handle[self.eva_database][self.variant_collection].find_one({'seq': 'GCA_000181335.3'}))
+        self.assertEqual(variant['contig'], 'CM001381.2')
+        variant = (self.connection_handle[self.eva_database][self.variant_collection].find_one(
```

Comment:
I was going to say that the parenthesis around the `self.connection_handle` calls are superfluous  but then I checked my code from EVA-2068 and realised I did the same thing :sweat: :sweat: